nc_test4/tst_varsperf.c

```diff
@@ -8,6 +8,9 @@
   under windows because of the use
   of gettimeofday().
 
+  WARNING: This test can only be run manually
+  and should not be run as part of the testsuite.
```

Comment:
Why is this the case? Seems like it can run just fine.

Reply:
@DennisHeimbigner can you clarify?